src/utils/SolutionsUtils.js

```diff
@@ -62,7 +62,7 @@ const patchSolutionIfLocalConfigExists = async (originalSolution) => {
 };
 
 const checkParametersValidationConstraintsInSolution = (data) => {
-  const parametersWithConstraint = data?.parameters?.filter((parameter) => parameter.options?.validation);
+  const parametersWithConstraint = data?.parameters?.filter((parameter) => parameter && parameter.options?.validation);
```

Comment:
just a question: optional chaining operator only (parameter?.options?.validation) won't work in this case ? :slightly_smiling_face:

Reply:
Nice catch, I changed it to use the optional chaining operator. Thank you for the suggestion :pray: